clusterpools/apply.sh

```diff
@@ -654,8 +654,12 @@ if [[ "$YQ_INSTALLED" != "false" ]]; then
                     read selection
                     if [ "$selection" -lt "$i" ]; then
                         CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME=${secret_names[$(($selection-1))]}
+                        CLUSTERPOOL_INSTALL_CONFIG_SECRET=$CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME
+                        CLUSTERPOOL_INSTALL_CONFIG_FILE=./$CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME.yaml
+                        oc get secret -n $CLUSTERPOOL_TARGET_NAMESPACE $CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME -o json | jq -r '.data["install-config.yaml"]' | ${BASE64} -d > $CLUSTERPOOL_INSTALL_CONFIG_FILE
```

Comment:
- Why are we setting `CLUSTERPOOL_INSTALL_CONFIG_SECRET=$CLUSTERPOOL_INSTALL_CONFIG_SECRET_NAME` and then not using `CLUSTERPOOL_INSTALL_CONFIG_SECRET` at all?
- This should also be `base64 --decode`